src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -186,18 +186,28 @@ export default class PhotoField extends React.Component {
         };
       }
     }
+
+    this.setState({ warningMessage: '' });
+  }
+
+  onZoomSliderChange = (e) => {
+    this.refs.cropper.zoomTo(e.target.value);
   }
 
   onZoom = (e) => {
     const zoomValue = e.detail.ratio;
     if (zoomValue < MAX_RATIO && zoomValue > MIN_RATIO) {
-      let warningMessage = null;
-      if (zoomValue >= WARN_RATIO) {
-        warningMessage = 'If you zoom in this close, your ID photo will be less clear.';
```

Comment:
Consolidated this with the rest of the warning message logic.

Reply:
Much tidier, well done.